prow/plugins/label/label_test.go

```diff
@@ -390,18 +523,37 @@ func TestLabel(t *testing.T) {
 	}
 
 	for _, tc := range testcases {
-		fakeClient, ice := getFakeRepo(tc.body, tc.commenter, tc.repoLabels, tc.issueLabels)
-		if err := handle(fakeClient, logrus.WithField("plugin", pluginName), ice); err != nil {
-			t.Errorf("For case %s, didn't expect error from label test: %v", tc.name, err)
-			continue
-		}
+		for i := 0; i < 3; i++ {
```

Comment:
Rather than using an int index here, I suggest using a list of functions `getFakeRepoIssueComment`, `getFakeRepoIssue`, `getFakeRepoPullRequest`.